lib/unpublish.js

```diff
@@ -37,10 +37,10 @@ function unpublish({ uri, url }) {
     url: '',
     scheduledTime: null
   }).then(function (resp) {
-    log('info', 'unpublished page updated:', uri);
+    log('info', `unpublished page updated: ${uri}`);
     return resp;
   }).catch(function (err) {
-    log('error', 'error update page list for unpublish', err.stack);
+    log('error', 'error update page list for unpublish', { error: err });
```

Comment:
you're logging errors as a separate object in some places, and logging the error stack in inline strings on other places. What's the best practice?

Reply:
I'm changing to use this method. It would make the whole error searchable when moved into ELK.

Reply:
Cool 👍